servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy

```diff
@@ -206,6 +207,18 @@ final class ServiceTalkLibraryPlugin extends ServiceTalkCorePlugin {
     }
   }
 
+  private static void applyPitestPlugin(Project project) {
+    project.configure(project) {
+      pluginManager.apply("info.solidsoft.pitest")
+
+      pitest {
+        timestampedReports = false
+        outputFormats = ['XML', 'HTML']
```

Comment:
spotbugs uses xml for CI and html for non-CI builds, see `applySpotBugsPlugin`. there is no need to have both